felix/proto/felixbackend.proto

```diff
@@ -236,8 +236,13 @@ message ActivePolicyRemove {
 }
 
 message PolicyID {
-  string tier = 1;
+  // The "tier" field has been deprecated and removed.
+  reserved 1;
+  reserved "tier";
```

Comment:
Do we need to reserve these, given that we do not support version skew in Felix?
I'm working on removing duplicate `HostMetadata` messages, and this is a question that applies also there.